torchtitan/config_manager.py

```diff
@@ -311,8 +311,20 @@ def __init__(self):
                 The schedule must be compatible with the split points and stages_per_rank.
 
                 Looped schedules (e.g. Interleaved1F1B) require specifying pipeline_parallel_degree = number of ranks,
-                and split_points = number of stages - 1""",
+                and split_points = number of stages - 1
+                """,
         )
+        self.parser.add_argument(
+            "--experimental.pipeline_parallel_schedule_csv",
+            type=str,
+            default="",
+            help="""
+                Specify the path to the pipeline parallel schedule csv file to use.
+                The pipeline_parallel_schedule argument must be either
+                PipelineScheduleSingle or PipelineScheduleMulti.
```

Comment:
nit: why do we not support PipelineScheduleRuntime?  (Is there a plan to unify these?)

Reply:
good point let me add that (it should work). In a follower up PR I can add a flag to use the runtime with the default schedules as well?

Reply:
Yea, that might be worth doing for benchmark purposes but I would rather it be short lived and simplify it to just use the runtime for all the schedules.